docs/gunicorn_ext.py

```diff
@@ -18,6 +18,14 @@
 used in the configuration file. The command line arguments are listed as well
 for reference on setting at the command line.
 
+.. note::
+
+    Settings can be specified by using environment variable
+    ``GUNICORN_CMD_ARGS``. All avaiable command line arguments can be used.
+    You may set multiple parameters by using space as separator. For example,
```

Comment:
"You may set multiple parameters by using space as separator." is not needed in my opinion.